pkg/mount/mountutil.go

```diff
@@ -110,3 +116,28 @@ func getDiskDevPath(partition string) (string, error) {
 	}
 	return "/dev/" + diskName, nil
 }
+
+// getPartitionName gets the partition name from the mountpoint. Each line of a mounts file
+// is passed to the function, which is parsed and partition name is obtained
+func (m *MountUtil) getPartitionName(mountLine string) (DeviceMountAttr, bool) {
+	mountAttr := DeviceMountAttr{}
+	isValid := false
+	if parts := strings.Split(mountLine, " "); parts[1] == m.mountPoint {
```

Comment:
nit: can we have length check to avoid npe

Reply:
Thanks @imazik . Fixed.